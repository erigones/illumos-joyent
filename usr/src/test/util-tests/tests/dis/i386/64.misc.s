/*
 * This file and its contents are supplied under the terms of the
 * Common Development and Distribution License ("CDDL"), version 1.0.
 * You may only use this file in accordance with the terms of version
 * 1.0 of the CDDL.
 *
 * A full copy of the text of the CDDL should have accompanied this
 * source.  A copy of the CDDL is also available via the Internet at
 * http://www.illumos.org/license/CDDL.
 */

/*
 * Copyright 2024 Oxide Computer Company
 */

/*
 * Test misc. instructions.
 */

.text
.align 16
.globl libdis_test
.type libdis_test, @function
libdis_test:
	rdpid	%rax
	rdpid	%r10
.size libdis_test, [.-libdis_test]
