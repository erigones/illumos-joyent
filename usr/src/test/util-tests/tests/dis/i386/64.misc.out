    libdis_test:     f3 0f c7 f8        rdpid  %rax
    libdis_test+0x4: f3 41 0f c7 fa     rdpid  %r10
